mathisfun/Complex.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+class Complex {
+public:
+
+	Complex();
```

Comment:
What does this CTOR do? Creates the number 0+0i?
Creates the number 17+35.7i?
Crashes the system?

If your code isn't self-explanatory, document it!